phssh_connector.py

```diff
@@ -984,16 +960,16 @@ def _block_ip(self, param):
             else:
                 port = "-dport {}".format(remote_port)
             no_port = False
-        except:
+        except Exception:
             port = ""
 
         try:
-            comment = "-m comment --comment '{} -- Added by Phantom'".format(param[SSH_JSON_COMMENT])
-        except:
+            comment = "-m comment --comment '{} -- Added by Phantom'".format(param.get(SSH_JSON_COMMENT))
```

Comment:
As the comment parameter is optional, param.get() will return None in case of an empty comment. This will send the comment as "None -- Added by Phantom".
Add condition to check the param, and remove try...except block.